src/main/java/pageobject/LoginPage.java

```diff
@@ -1,79 +1,93 @@
 package pageobject;
 
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
+//import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.remote.server.handler.ClickElement;
+//import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+import Base.BaseActions;
 
-public class LoginPage {
 
-	
-	@FindBy(id="user_email")
-	WebElement email;
-	
-	@FindBy(id="user_password")
-	WebElement password;
+public class LoginPage extends BaseActions
+{
 
-	@FindBy(name="commit")
-	WebElement submit;
+	WebDriver  driver;
 	
-	@FindBy(xpath="//div[contains(@class,'alert-danger')]")
-	WebElement errorMsg;
+	By home_login = By.xpath("//button[@class='a-button f-b pure-input-1 f-b-4A90E2-a f-b-4A90E2-h f-border-4A90E2 f-c-FFFFFF-a f-c-FFFFFF-h f-c-4A90E2 f-l-38 f-m-a f-r-20 f-s-16 f-w-90 f-600']");
```

Comment:
Using multiple class names in xpath is not recommended as the class can be changed.

Reply:
Changed Xpath